scripts/gen-dumbfs.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+##############################################################################
+##
+##  This file is part of the Chaos Kernel, and is made available under
+##  the terms of the GNU General Public License version 2.
+##
+##  Copyright (C) 2017 - Jakob Kellendonk <jakob.kellendonk@epitech.eu>
+##
+##############################################################################
+
+if (( $# != 2 ))
+then
+	echo -E "usage: ./${0} IN_DIR OUT_FILE"
+	echo -E "[DUMBFS_GEN]	Missing args, exiting."
+	exit 1
+fi
+
+IN_DIR=${1}
+OUT_FILE=${2}
+
+shopt -s nullglob
+
+>${OUT_FILE}
```

Comment:
I think it's more standard to use `touch ${OUT_FILE}`

Reply:
Touch isn't a built-in, so technically an external dependency? I can change this if it's necessary.

Reply:
Touch is a coreutils, it's safe to assume it will always be present.